### PR TITLE
Ale 0.1.1

### PR DESCRIPTION
**Pull request by darker0n**

### CODE REVIEW

File: Library/Formula/ale.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Ale < Formula
+  homepage "https://darker0n.github.io/ale"
+  url "https://github.com/darker0n/ale/archive/v0.1.1.tar.gz"
+  version "0.1.1"
```
Comment (DomT4) [inline Library/Formula/ale.rb]: Unnecessary. Brew can read this from the URL in this case.

File: Library/Formula/ale.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "2bae9392f02dc6fc1a77c637fa0139a26648b0303a63eae9b46661999985c679"
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec
```
Comment (DomT4) [inline Library/Formula/ale.rb]: ` ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages" ` (Presuming it needs 2.7, of course).

File: Library/Formula/ale.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    libexec.install Dir["core"]
+    bin.install "ale.py"
+
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    system "python", "setup.py", "install", "--user"
```
Comment (DomT4) [inline Library/Formula/ale.rb]: ` system "python", *Language::Python.setup_install_args(libexec) `

File: Library/Formula/ale.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    system "alias", "ale=\"#{prefix}/bin/ale.py\""
```
Comment (DomT4) [inline Library/Formula/ale.rb]: What does this gain us?

File: Library/Formula/ale.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+class Ale < Formula
+  homepage "https://darker0n.github.io/ale"
+  url "https://github.com/darker0n/ale/archive/v0.1.1.tar.gz"
+  version "0.1.1"
+  sha256 "2bae9392f02dc6fc1a77c637fa0139a26648b0303a63eae9b46661999985c679"
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec
+
+    libexec.install Dir["core"]
```
Comment (DomT4) [inline Library/Formula/ale.rb]: Either ` libexec.install Dir["core/*"] ` or ` libexec.install "core" `.

**Comment by darker0n** [comment]
Ok, thanks, but ale can't import files from core library:
    
    Traceback (most recent call last):
     File "/Users/darker0n/workspace/homebrew/Cellar/ale/0.1.1/libexec/bin/ale.py", line 3, in <module>
     from core import wcolors
     ImportError: No module named core

**Comment by mikemcquaid** [comment]
This is a self-submission of a project that's not yet popular and less than a week old. Passing on this for now, sorry! I suggest you host it yourself in a tap.

**Comment by DomT4** [comment]
> Ok, thanks, but ale can't import files from core library:

This is probably because you didn't set up the ` PYTHONPATH ` properly. See [this document](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md) for help if you set up a tap yourself.

**Pull request closed without merging**